
CommandNotFoundError: Your shell has not been properly configured to use 'conda activate'.
To initialize your shell, run

    $ conda init <SHELL_NAME>

Currently supported shells are:
  - bash
  - fish
  - tcsh
  - xonsh
  - zsh
  - powershell

See 'conda init --help' for more information and options.

IMPORTANT: You may need to close and restart your shell after running 'conda init'.


cat: write error: Broken pipe
2023-01-26 21:14:25.148052
2023-01-26 21:14:25.613176
2023-01-26 21:14:26.033198

real	0m1.821s
user	0m1.638s
sys	0m0.153s
cat: write error: Broken pipe
2023-01-26 21:14:37.239103
2023-01-26 21:14:37.637878
2023-01-26 21:14:38.034652

real	0m1.706s
user	0m1.535s
sys	0m0.148s
cat: write error: Broken pipe
2023-01-26 21:14:49.229537
2023-01-26 21:14:49.605297
2023-01-26 21:14:49.996084

real	0m1.663s
user	0m1.452s
sys	0m0.181s
cat: write error: Broken pipe
2023-01-26 21:15:01.141241
2023-01-26 21:15:01.498794
2023-01-26 21:15:01.892312

real	0m1.610s
user	0m1.440s
sys	0m0.148s
cat: write error: Broken pipe
2023-01-26 21:15:13.323532
2023-01-26 21:15:14.616115
2023-01-26 21:15:15.088564

real	0m2.917s
user	0m2.648s
sys	0m0.243s
cat: write error: Broken pipe
2023-01-26 21:15:26.411226
2023-01-26 21:15:27.302267
2023-01-26 21:15:28.372823

real	0m2.980s
user	0m2.761s
sys	0m0.194s
cat: write error: Broken pipe
2023-01-26 21:15:39.544263
2023-01-26 21:15:40.492710
2023-01-26 21:15:41.927107

real	0m3.269s
user	0m3.003s
sys	0m0.241s
cat: write error: Broken pipe
2023-01-26 21:15:53.111243
2023-01-26 21:15:53.360884
2023-01-26 21:15:53.485720

real	0m1.269s
user	0m1.119s
sys	0m0.130s
mkdir: cannot create directory ‘SCODE_D_2’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_4’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_6’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
mkdir: cannot create directory ‘SCODE_D_8’: File exists
┌ Warning: `CSV.read(input; kw...)` is deprecated in favor of `using DataFrames; CSV.read(input, DataFrame; kw...)
│   caller = ip:0x0
└ @ Core :-1
┌ Warning: Assignment to `WZ` in soft scope is ambiguous because a global variable by the same name exists: `WZ` will be treated as a new local. Disambiguate by using `local WZ` to suppress this warning or `global WZ` to assign to the existing global variable.
└ @ ~/stephenz/locaTE-paper/tools/SCODE/SCODE.jl:69
┌ Warning: `writeheader=false` is deprecated in favor of `header=false`
│   caller = ip:0x0
└ @ Core :-1
a
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:18: UserWarning:  WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.
  if expression==None: warnings.warn(" WARNING: No expression data argument given. if you intend to load the data from a file, call the method 'read_expression_file'.")
/data/gpfs/projects/punim0638/stephenz/miniconda3/lib/python3.10/site-packages/Scribe-0.0.0-py3.10.egg/Scribe/causal_network.py:298: FutureWarning: Could not cast to int64, falling back to object. This behavior is deprecated. In a future version, when a dtype is passed to 'DataFrame', either all columns will be cast to that dtype, or a TypeError will be raised.
  max_rdi_delay = pd.DataFrame({node_id: [np.nan for i in self.node_ids] for node_id in self.node_ids}, index=self.node_ids,
Create causal_model successfully
  Activating project at `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-LI/dyn-LI-1000-7/pidc_env`
    Updating registry at `~/.julia/registries/General.toml`
   Resolving package versions...
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-LI/dyn-LI-1000-7/pidc_env/Project.toml`
  No Changes to `/data/gpfs/projects/punim0638/stephenz/locaTE-paper/data/simulated/Synthetic_1000/dyn-LI/dyn-LI-1000-7/pidc_env/Manifest.toml`
Getting nodes...
Inferring network...
